crates/verifier/src/plonk/converter.rs

```diff
@@ -118,6 +118,12 @@ pub(crate) fn load_plonk_proof_from_bytes(
     buffer: &[u8],
     num_bsb22_commitments: usize,
 ) -> Result<PlonkProof, PlonkError> {
+    if buffer.len()
+        < 384 + 5 * 32 + 96 + 128 + num_bsb22_commitments * 32 + num_bsb22_commitments * 64
```

Comment:
There should definitely be constants that are used here rather than the hardcoded numbers